### PR TITLE
License the work under CC BY 4.0

### LINKED ISSUE #162: license
what is it and can a license file be added?

### PR DESCRIPTION
**Pull request by pirapira**
This is not mergeable without at least @gavofyork's approval.  This would fix #162.  My choice of the license is tentative and arbitrary.

### CODE REVIEW

**Comment by axic** [comment]
Perhaps it would make sense collecting approval here from all the authors?

I agree to CC BY 4.0 for my contributions.

**Comment by pirapira** [comment]
I agree to CC BY 4.0 for my contributions.

**Comment by chriseth** [comment]
I agree to CC BY 4.0 for my contributions.

**Comment by chriseth** [comment]
From what I have gathered about licensing, and looking at the github contributors page, approvals from @gavofyork @CJentzsch and @Gustav-Simonsson should be sufficient. Simple spelling corrections are not subject to copyright.

**Comment by jamesray1** [comment]
There's a discussion on this here: https://gitter.im/ethereum/yellowpaper?at=5a4d72e368d092bb621f120a.

**Comment by CJentzsch** [comment]
I agree to CC BY 4.0 for my contributions.

**Comment by jamesray1** [comment]
Thanks @CJentzsch! One down, two to go.

File: Paper.tex [hunk 1]
```diff
@@ -1156,7 +1156,7 @@ \section{Acknowledgements}
 
 \section{Availability}
 
-The source of this paper is maintained at \url{https://github.com/ethereum/yellowpaper/}. An auto-generated PDF is located at \url{https://ethereum.github.io/yellowpaper/paper.pdf}.
+The source of this paper is maintained at \url{https://github.com/ethereum/yellowpaper/} and is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit \url{http://creativecommons.org/licenses/by/4.0/}. An auto-generated PDF is located at \url{https://ethereum.github.io/yellowpaper/paper.pdf}.
```
Comment (5chdn) [inline Paper.tex]: If you change it to Creative Commons Attribution **Share-Alike** 4.0 this will be in line with #402 and can be merged easily.

Comment (pirapira) [reply Paper.tex]: Right, yes, I can do that.  https://creativecommons.org/faq/#can-i-combine-material-under-different-creative-commons-licenses-in-my-work

**Comment by pirapira** [comment]
On the other hand, the `master` branch already contains a license.  So there is not much merit in pursuing this thread.  Thanks everyone for the agreements so far.

**Pull request closed without merging**